### PR TITLE
chore: add Chinese translation

### PR DESCRIPTION
**Pull request by xuefeng-bai**
### SUMMARY

To add/update Chinese translation.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by zhaoyongjie** [comment]
@xuefeng-bai I think you cannot merge the PR into a release branch(1.5). Do you mind that change it to the master branch, or let me switch it to the master branch?

File: superset/translations/zh/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -6673,7 +6671,7 @@ msgstr "一月"
 msgid ""
 "Json list of the column names that should be read. If not None, only "
 "these columns will be read from the file."
-msgstr ""
+msgstr "应读取的列名的Json列表。如果没有，则仅从文件中读取这些列。"
```
[the remark this answers is not in the document]
Comment (xuefeng-bai) [reply superset/translations/zh/LC_MESSAGES/messages.po:6674]: 你能帮我处理吗，这次的操作有些复杂，我第一次操作还不太熟悉，未来改动的提交我会直接到master。另外还有其他提交规则需要注意的吗？方便后续我更快更好的完成PR

[the remark this answers is not in the document]
Comment (zhaoyongjie) [reply superset/translations/zh/LC_MESSAGES/messages.po:6674]: 看起来有很多冲突.

**Comment by dpgaspar** [comment]
@xuefeng-bai could you please rebase and fix the conflicts?

cc: @rusackas should we close this one?

**Comment by zhaoyongjie** [comment]
> @xuefeng-bai could you please rebase and fix the conflicts?
> 
> cc: @rusackas should we close this one?

Close this PR first. When the conflicts resolved, please ping me to reopen it. Thanks for the contribution. @xuefeng-bai

**Pull request closed without merging**